<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="7,3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Slider和QColor</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Red</string>
        </property>
        <property name="buddy">
         <cstring>hSliderRed</cstring>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Green</string>
        </property>
        <property name="buddy">
         <cstring>hSliderGreen</cstring>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Blue</string>
        </property>
        <property name="buddy">
         <cstring>hSliderBlue</cstring>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Alpha</string>
        </property>
        <property name="buddy">
         <cstring>hSliderAlpha</cstring>
        </property>
       </widget>
       <widget class="QSlider" name="hSliderRed">
        <property name="geometry">
         <rect>
          <x>71</x>
          <y>31</y>
          <width>151</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="hSliderGreen">
        <property name="geometry">
         <rect>
          <x>71</x>
          <y>57</y>
          <width>151</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="hSliderBlue">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>90</y>
          <width>151</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="hSliderAlpha">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>120</y>
          <width>151</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>30</y>
          <width>181</width>
          <height>111</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>水平</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSlider" name="hSliderPing">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollBar" name="hScrollBar">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBarH">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Dial和LCDNumber</string>
       </property>
       <widget class="QDial" name="dial">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>101</width>
          <height>91</height>
         </rect>
        </property>
        <property name="notchesVisible">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>40</y>
          <width>121</width>
          <height>121</height>
         </rect>
        </property>
        <property name="digitCount">
         <number>7</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>30</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>LCD显示进制</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="rBtnDec">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>60</y>
          <width>115</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>十进制</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="rBtbBin">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>90</y>
          <width>115</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>二进制</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="rBtnOct">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>120</y>
          <width>115</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>八进制</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="rBtnHex">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>150</y>
          <width>115</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>十六进制</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="5,1,3,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>垂直</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSlider" name="verticalSlider">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollBar" name="verticalScrollBar">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="invertedAppearance">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBarV">
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnExit">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>hSliderPing</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hScrollBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hScrollBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBarH</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBarV</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalScrollBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>564</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>488</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rBtnDec</sender>
   <signal>clicked()</signal>
   <receiver>lcdNumber</receiver>
   <slot>setDecMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>488</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rBtbBin</sender>
   <signal>clicked()</signal>
   <receiver>lcdNumber</receiver>
   <slot>setBinMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>488</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rBtnHex</sender>
   <signal>clicked()</signal>
   <receiver>lcdNumber</receiver>
   <slot>setHexMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>547</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>488</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rBtnOct</sender>
   <signal>clicked()</signal>
   <receiver>lcdNumber</receiver>
   <slot>setOctMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>517</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>488</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnExit</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
